storagedriver/s3/s3.go

```diff
@@ -195,7 +209,7 @@ func New(params DriverParameters) (*Driver, error) {
 	// 	}
 	// }
 
-	return &Driver{s3obj, bucket, params.Encrypt, params.RootDirectory}, nil
+	return &Driver{s3obj, bucket, params.ChunkSize, params.Encrypt, params.RootDirectory}, nil
```

Comment:
This struct should have keys called out. I'm surprised this passed the linter and vet.